frontend/src/components/MainPage/Diary/WorkoutModal.tsx

```
@@ -0,0 +1,81 @@
+      .then((data) => setExercises(data.exercises))
+      .catch((error) => console.error("Error fetching workouts:", error));
+  }, [selectedWorkout]);
+
+  return (
+    <>
+      <p className="text-lg font-semibold p-2">
+        Workout {selectedWorkout.workout_id}
+      </p>
+      {exercises.length > 0 ? (
+        exercises.map((exercise) => (
+          <div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)